scripts/validate-build.sh

```
@@ -0,0 +1,483 @@
+        return 1
+    fi
+
+    log_success "remoteEntry.js present and valid ($SIZE bytes)"
+
+    # Check for missing webpack chunks
+    log_info "Checking for webpack chunks..."
+    CHUNK_REFS=$(grep -o 'chunk[0-9]\+' /tmp/dist-check/remoteEntry.js || true)
+    MISSING_CHUNKS=0
+
+    for chunk in $CHUNK_REFS; do
+        if ! ls /tmp/dist-check/*.bundle.js 2>/dev/null | grep -q "$chunk"; then
```

Shellcheck (0.11.0)

[warning] Don't use ls | grep. Use a glob or a for loop with a condition to allow non-alphanumeric filenames.

(SC2010)